File: app/src/main/java/com/janyo/janyoshare/classes/InstallApp.java
package com.janyo.janyoshare.classes;

import android.graphics.drawable.Drawable;

public class InstallApp
{
	private String name;
	private String versionName;
	private String sourceDir;
	private String packageName;
	private Drawable icon;

	public String getName()
	{
		return name;
	}

	public void setName(String name)
	{
		this.name = name;
	}

	public String getVersionName()
	{
		return versionName;
	}

	public void setVersionName(String versionName)
	{
		this.versionName = versionName;
	}

	public String getSourceDir()
	{
		return sourceDir;
	}

	public void setSourceDir(String sourceDir)
	{
		this.sourceDir = sourceDir;
	}

	public String getPackageName()
	{
		return packageName;
	}

	public void setPackageName(String packageName)
	{
		this.packageName = packageName;
	}

	public Drawable getIcon()
	{
		return icon;
	}

	public void setIcon(Drawable icon)
	{
		this.icon = icon;
	}
}

File: app/src/main/java/com/janyo/janyoshare/classes/InstallApp.kt
package com.janyo.janyoshare.classes

import android.graphics.drawable.Drawable
import java.io.Serializable

class InstallApp : Serializable
{
	var name: String? = null
	var versionName: String? = null
	var versionCode: Int = 0
	var sourceDir: String? = null
	var packageName: String? = null
	var iconPath: String? = null
	var icon: Drawable? = null
	var size: Long = 0
	var installTime: Long = 0
	var updateTime: Long = 0
}

/**
 * N——appName
 * V——versionName
 * W——versionCode
 * P——packageName
 */
